scripts/cdf.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+import warnings
+
+#Put file you want to read in input folder
+
+#**********************************************************************
+#List to store the data (ADD AS MUCH AS NECESSARY)
+#**********************************************************************
+
+xlist = []
+ylist = []
+
+#**********************************************************************
+#File Read
+#fname : put the name of the file you are reading 
+#**********************************************************************
+
+
+#Change path 
+df = pd.read_csv('sample/log-rps-100-iv-10.csv')
```

Comment:
instead of hard coding the file name here, make it be a command line argument when you run the script with something like: `python cdf.py sample/mydatafile.csv` or `python cdf.py -f sample/mydatafile.csv` (if using a helper library like getopt)

More info here: https://www.digitalocean.com/community/tutorials/python-command-line-arguments